ecoscope/io/earthranger.py

```diff
@@ -29,6 +29,7 @@ def __init__(self, sub_page_size=4000, tcp_limit=5, **kwargs):
 
         self.sub_page_size = sub_page_size
         self.tcp_limit = tcp_limit
+        self._display_map = None
```

Comment:
would it make sense to kick this off everytime the client is instantiated?

Reply:
Not on the synchronous client - the fetch simply takes too long.
Async however...maybe